src/main/groovy/com/deploygate/gradle/plugins/internal/credentials/CliCredentialStore.groovy

```diff
@@ -0,0 +1,82 @@
+package com.deploygate.gradle.plugins.internal.credentials
+
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import org.jetbrains.annotations.NotNull
+import org.jetbrains.annotations.Nullable
+import org.jetbrains.annotations.VisibleForTesting
+
+class CliCredentialStore {
+    private static final Gson GSON = new Gson()
```

Comment:
Use gson instead of Groovy's json parser because we should avoid unmeaningful dynamic types.